src/test/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/AssemblyBasedSVDiscoveryTestDataProviderForSimpleSV.java

```diff
@@ -0,0 +1,1032 @@
+    }
+
+    public static final boolean testDataInitialized;
+    public static final TestDataForSimpleSV forSimpleDeletion_plus;
+    public static final TestDataForSimpleSV forSimpleDeletion_minus;
+    public static final TestDataForSimpleSV forDeletionWithHomology_plus;
+    public static final TestDataForSimpleSV forDeletionWithHomology_minus;
+    public static final TestDataForSimpleSV forSimpleInsertion_plus;
+    public static final TestDataForSimpleSV forSimpleInsertion_minus;
+    public static final TestDataForSimpleSV forLongRangeSubstitution_fudgedDel_plus;
+    public static final TestDataForSimpleSV forLongRangeSubstitution_fudgedDel_minus;
+    public static final TestDataForSimpleSV forLongRangeSubstitution_fatIns_plus;
+    public static final TestDataForSimpleSV forLongRangeSubstitution_fatIns_minus;
+    public static final TestDataForSimpleSV forLongRangeSubstitution_DelAndIns_plus;
+    public static final TestDataForSimpleSV forLongRangeSubstitution_DelAndIns_minus;
+    public static final TestDataForSimpleSV forSimpleTanDupContraction_plus;
+    public static final TestDataForSimpleSV forSimpleTanDupContraction_minus;
+    public static final TestDataForSimpleSV forSimpleTanDupExpansion_ins_plus;
+    public static final TestDataForSimpleSV forSimpleTanDupExpansion_ins_minus;
+    public static final TestDataForSimpleSV forSimpleTanDupExpansion_dup_plus;
+    public static final TestDataForSimpleSV forSimpleTanDupExpansion_dup_minus;
+    public static final TestDataForSimpleSV forSimpleTanDupExpansionWithNovelIns_ins_plus;
+    public static final TestDataForSimpleSV forSimpleTanDupExpansionWithNovelIns_ins_minus;
+    public static final TestDataForSimpleSV forSimpleTanDupExpansionWithNovelIns_dup_plus;
+    public static final TestDataForSimpleSV forSimpleTanDupExpansionWithNovelIns_dup_minus;
+    public static final TestDataForSimpleSV forComplexTanDup_1to2_pseudoHom_plus;
+    public static final TestDataForSimpleSV forComplexTanDup_1to2_pseudoHom_minus;
+    public static final TestDataForSimpleSV forComplexTanDup_2to1_pseudoHom_plus;
+    public static final TestDataForSimpleSV forComplexTanDup_2to1_pseudoHom_minus;
+    public static final TestDataForSimpleSV forComplexTanDup_3to2_noPseudoHom_plus;
+    public static final TestDataForSimpleSV forComplexTanDup_3to2_noPseudoHom_minus;
+    public static final TestDataForSimpleSV forComplexTanDup_2to3_noPseudoHom_plus;
+    public static final TestDataForSimpleSV forComplexTanDup_2to3_noPseudoHom_minus;
+    public static final TestDataForSimpleSV forComplexTanDup_1to2_short_pseudoHom_plus;
+    public static final TestDataForSimpleSV forComplexTanDup_1to2_short_pseudoHom_minus;
+    public static final TestDataForSimpleSV forComplexTanDup_2to3_short_noPseudoHom_plus;
+    public static final TestDataForSimpleSV forComplexTanDup_2to3_short_noPseudoHom_minus;
+
+    static {
+        try ( final ByteArrayOutputStream outputStream = new ByteArrayOutputStream() ){
```

Comment:
Why is there a single `outputStream` shared by all the tests? It looks like most of the tests call reset on it before writing anything to it. Why have a shared one at all -- why not just do it in each test?

Also, do you need output streams at all? It looks like you are just using them to concatenate the portions of the chimera (flanks and homology). Why not just use string concatentation?